examples/assets-and-custom-transfer-logic/assets/clawback-escrow.teal

```diff
@@ -0,0 +1,116 @@
+#pragma version 2
```

Comment:
isn't this file same as `clawback-escrow.py` ?

Reply:
yes, it's only for reference purpose. Should i remove?

Reply:
yes, I think this is intermediate file, removing seems better.

Reply:
Ok, removing.